src/rpcmisc.cpp

```diff
@@ -49,12 +49,13 @@ UniValue getinfo(const UniValue& params, bool fHelp)
             "{\n"
```

Comment:
Lol, how did I miss that? I added localservices in 99ddc6cb706e825e54da244a10d4d6389fc5eaae

@laanwj Thanks for catching that!